src/app/clusters/thermostat-server/thermostat-server.cpp

```diff
@@ -766,7 +757,7 @@ CHIP_ERROR ThermostatAttrAccess::Read(const ConcreteReadAttributePath & aPath, A
     }
     break;
     case ActivePresetHandle::Id: {
-        Delegate * delegate = GetDelegate(aPath.mEndpointId);
+        auto delegate = GetDelegate(aPath.mEndpointId);
```

Comment:
Revert these type changes to keep the diff small?

Reply:
Or did you need to change anyway because of Thermostat:: namespace requirement?

Reply:
Yeah, this is the actual namespace conflict.